tox.ini

```diff
@@ -6,7 +6,7 @@ envlist =
        {py27,py32,py33,py34}-django{17,18,master}
 
 [testenv]
-commands = ./runtests.py --fast
+commands = ./runtests.py --fast {posargs}
```

Comment:
@xordoquy 
Possible use case: stop tests execution after the first error encountered for a specific tox environment.

```sh
$ tox -e py34-django18 -- -x
```
here `-x` is an argument intended for `py.test`